aws_lambda_powertools/utilities/parser/parser.py

```diff
@@ -17,7 +18,7 @@ def event_parser(
     handler: Callable[[Any, LambdaContext], EventParserReturnType],
     event: Dict[str, Any],
     context: LambdaContext,
-    model: Type[Model],
+    model: Type[Model] = None,
```

Comment:
As the `model` is now `Optional` with `None` as the default value, we need to change the parameter type

```suggestion
    model: Optional[Type[Model]] = None,
```